servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/GraphQLConfigurationProperties.kt

```diff
@@ -34,7 +34,8 @@ data class GraphQLConfigurationProperties(
     val playground: PlaygroundConfigurationProperties = PlaygroundConfigurationProperties(),
     val sdl: SDLConfigurationProperties = SDLConfigurationProperties(),
     val introspection: IntrospectionConfigurationProperties = IntrospectionConfigurationProperties(),
-    val batching: BatchingConfigurationProperties = BatchingConfigurationProperties()
+    val batching: BatchingConfigurationProperties = BatchingConfigurationProperties(),
+    val automaticPersistedQueriesEnabled: Boolean = false
```

Comment:
We might want to do nest config here instead. You will eventually want to control the default implemetation with some extra config like `ttl` so having a separate top level property for that would be annoying

Instead

```yml
graphql:
  automaticPersistedQueries:
    enabled: true
    # open for future fields here too
```